apps/keyboard/test/unit/imes/latin/worker_test.js

```diff
@@ -1,7 +1,7 @@
+/* jshint elision: true */
```

Comment:
Let's not rely on an option on jshint to allow a syntax that reduce the readability.
See the comments below.